docs/src/pages/discover-more/roadmap/roadmap.md

```diff
@@ -68,9 +68,10 @@ Here are the components we will work on being supported in the Material-UI ecosy
 | AutoComplete | 🛠 |
 | ComboBox | 🛠 |
 | MultiSelect | 🛠 |
+| Date Picker (& range) | 🛠 |
```

Comment:
"Picker" is pretty heavy jargon and hard to translate. The nav already uses "Date input". We should stick with either one in the future.

Reply:
I fear this question has been sealed for us in the past. People know to find this component using the "datepicker" query.

Reply:
Citation needed.

Also: All the more reason to use date input to lure these poor folks using input type="date"

Reply:
Here is what I have in mind, we have to match what people query, not improve the searched query: this list contains, I hope, most of the most popular date picker. It's part of the list (I have removed final products) I have used for my proposal in https://github.com/mui-org/material-ui-pickers/issues/1293#issuecomment-538374383

- https://baseweb.design/components/datepicker/
- https://material.angular.io/components/datepicker/overview
- http://react.carbondesignsystem.com/?path=/story/datepicker--range-with-calendar
- https://ant.design/components/date-picker/
- https://blueprintjs.com/docs/#datetime/datepicker
- https://bootstrap-datepicker.readthedocs.io/
- https://github.com/airbnb/react-dates
- https://jqueryui.com/datepicker/
- https://material.io/components/pickers/
- https://reactdatepicker.com/
- https://vuetifyjs.com/en/components/date-pickers
- https://www.telerik.com/kendo-react-ui/components/dateinputs/datepicker/

Reply:
> using input type="date"

I think that history has teach us that most of our users don't want to use it 🙃

Reply:
Why do you include links that don't include pickers and are not translated?

Reply:
> Here is what I have in mind, we have to match what people query, 

still looking for a citation here. You reverse this to support your claim when you should provide some evidence what users first think/query vs. how they arrive at a solution

Reply:
Where do we use "Date input"? I can't find references to it. What's a citation?